### DISCUSSION TITLE
关于小狼毫的界面窗口

### DISCUSSION THREAD

**Discussion by weique** [rime/home#1341] [General]
<img width="216" alt="Snipaste_2023-07-16_18-58-11" src="https://github.com/rime/home/assets/13903773/44fe3ee8-c737-42d8-9582-699d034dc721">

想把小狼毫的窗口界面调整到像红框那样。窗口宽度、高亮候选区域的宽度与提示拼音的宽度接近，而不是在右边多出一块。
试遍了相关参数，都调整不出最合适的效果。
希望有大佬能指点一下。

**Comment by fxliang** [comment]
这个后面修复了
https://github.com/rime/weasel/actions/runs/5505926894

**Reply by weique** [reply]
刚刚更新了，果然已经修复！
太棒了，感谢！
